src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -353,6 +385,42 @@ public static boolean postOMFBatch(String payload) {
     }
     return true;
   }
+  
+  /**
+   * Posts the OMF batch to OSIsoft Cloud Services.
+   *
+   * @param payload the payload to post
+   * @return returns the http response code
+   */
+  public static boolean postOcsBatch(String payload) {
+
+    String postHeaderType = "&messagetype=type";

```

Comment:
final?

Reply:
whoops, this one does change later for the data messages. need to repush

Reply:
this is fixed